styles/sass/pages/_contact-us.scss

```diff
@@ -258,7 +275,14 @@ a{
     }
 
     &__map--cropped img {
-      margin: 0;
+      margin: 0px 0px 0px -180px;
```

Comment:
if you will use just the left value you can use the property
`margin-left:` or `margin-+theDirection:`